client/src/www/ui_components/app-root.js

```diff
@@ -55,6 +54,49 @@ import {mixinBehaviors} from '@polymer/polymer/lib/legacy/class.js';
 import {html} from '@polymer/polymer/lib/utils/html-tag.js';
 import {PolymerElement} from '@polymer/polymer/polymer-element.js';
 
+function makeLookUpLanguage(availableLanguages) {
```

Comment:
Why is all this code being added?
Are you inline the l10n library?

If so, we need a TODO to move this to shared code, because I believe this is used by the Manager as well?